tests/test_features.py

```diff
@@ -433,3 +433,23 @@ def __make_data(coeffs, freq):
         freq = np.cumsum(np.abs(np.random.randn(1 + n_fft//2, 2)), axis=0)
         S = __make_data(coeffs, freq)
         yield __test, S, coeffs, freq
+
+
+def test_chroma_to_tonnetz():
```

Comment:
Do you have any pre-computed data that we could use to build regression tests, as well as the general sanity checks already in place?